support-frontend/assets/components/onboarding/appBadgesDownload/guardianAppsQrCode.tsx

```diff
@@ -0,0 +1,31 @@
+export function GuardianAppsQrCode() {
```

Comment:
would be easier to review if these were in a `.svg` file (although possibly more clunky at compile time)